experimental/devices/mfrc522/mfrc522.go

```diff
@@ -0,0 +1,732 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package mfrc522 controls a Mifare RFID card reader.
+//
+// The code is largely ported from Python library : https://github.com/mxgxw/MFRC522-python
```

Comment:
Is still the case? The goal of periph drivers is to not just be ports but really look at the datasheet and implement what matters.

Reply:
Ok, will remove :)